test/runtime/runtime-python/test/main.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import json
+from main.Address import Address
+from main.NestedObject import NestedObject
```

Comment:
Seems that the CI cannot access the models. Are you able to run the commands locally? You can check the error here: https://github.com/asyncapi/modelina/actions/runs/7635660479/job/20805641740?pr=1748

Reply:
` npm run test:runtime:python` this command is causing error : 
` > cross-env CI=true jest ./test/runtime/runtime-python.spec.ts`
`FAIL test/runtime/runtime-python.spec.ts`
`  ● Test suite failed to run`
`Bindings not found`

Reply:
You need to be able to run the scripts locally before the CI can run it as well 🙂

Reply:
@jonaslagoni , there were some issues in the testing script ,I resolved them , now the python testing script returns output  :
..
----------------------------------------------------------------------
Ran 2 tests in 0.000s

OK

But the exec command gets this output in stderr . I have been stuck on this part.

Reply:
Did you push the changes? 🤔

Reply:
pushed now

Reply:
@jonaslagoni  
There are certain problems :
1.The generated code which is added by generator contain two files, 
Address.py and NestedObject.py  
![Screenshot from 2024-02-07 16-23-07](https://github.com/asyncapi/modelina/assets/98508374/c1134f8b-80a9-451b-9cb6-294627300d44)

So when this file runs ,I get this error:
![Screenshot from 2024-02-07 16-25-21](https://github.com/asyncapi/modelina/assets/98508374/928e9ee4-4483-40fa-badf-7922c1da5769)

So when running locally I removed this import manually. 

2.I added a __init__.py file manually in the generated folder to make that as module so it can be imported.This thing is failing in the CI also .

Reply:
@jonaslagoni  
There are certain problems :
1.The generated code which is added by generator contain two files, 
Address.py and NestedObject.py  
![Screenshot from 2024-02-07 16-23-07](https://github.com/asyncapi/modelina/assets/98508374/c1134f8b-80a9-451b-9cb6-294627300d44)

So when this file runs ,I get this error:
![Screenshot from 2024-02-07 16-25-21](https://github.com/asyncapi/modelina/assets/98508374/928e9ee4-4483-40fa-badf-7922c1da5769)

So when running locally I removed this import manually. 

2.I added a __init__.py file manually in the generated folder to make that as module so it can be imported.This thing is failing in the CI also .

Reply:
> So when running locally I removed this import manually.

You are not allowed to manually make changes to the generated models (cause the manual changes will be overwritten next time you run the command). 

You should setup the project so that the test files and other project files i.e. init.py etc are added manually, but the generated models are used as is with no other manual changes needed.

Is this not possible with the current generated models?

> 2.I added a init.py file manually in the generated folder to make that as module so it can be imported.This thing is failing in the CI also .

This has not been committed?

Reply:
@jonaslagoni  The whole main folder is getting generated by the script so how can I add init.py manually?
The first screenshot which I attached is totally associates with the generated code and it give import errors,So I am not getting what can I do for that

Reply:
You can manually add the main folder and the init script as you see fit 👍 The script just generates the models next to it.

Reply:
@jonaslagoni 
I have added some fixes, so the script should run now . 
1.Python script is returning OK ran 2 tests but the script is getting it as error.